packages/toolkit/README.md

```diff
@@ -48,6 +48,10 @@ The Redux Toolkit docs are available at **https://redux-toolkit.js.org**, includ
 
 The Redux core docs at https://redux.js.org includes the full Redux tutorials, as well usage guides on general Redux patterns.
 
+## Contributing
```

Comment:
re-reading this, i think it should be moved to the end of the readme, after the sections about what it's for and what it includes